### PR TITLE
sort imports

### PR DESCRIPTION
**Pull request by devoncarew**
Sort some imports, and remove one analysis warnings re: naming a local variable (`Q` is not lowerCamelCase).

### CODE REVIEW

File: packages/flx/lib/signing.dart [hunk 1]
```diff
@@ -165,8 +165,8 @@ ECSignature _asn1ParseSignature(Uint8List signature) {
 }
 
 PublicKey _publicKeyFromPrivateKey(ECPrivateKey privateKey) {
-  ECPoint Q = privateKey.parameters.G * privateKey.d;
-  return new ECPublicKey(Q, privateKey.parameters);
+  ECPoint q = privateKey.parameters.G * privateKey.d;
```
Comment (Hixie) [inline packages/flx/lib/signing.dart]: i funny think we should make the change. is the name of the variable in the spec we're implementing, as I understand it.

Comment (devoncarew) [inline packages/flx/lib/signing.dart]: I can change this to an //ignore

Comment (devoncarew) [inline packages/flx/lib/signing.dart]: Hmm, I reverted this, re-analyzed the source, and am no longer seeing the lint.

Comment (Hixie) [inline packages/flx/lib/signing.dart]: @pq was working on making that lint not fire for single-character variables of any case, and the `flutter analyze` command has a hack to hide this specific warning, FWIW.

**Comment by Hixie** [comment]
remaining changes: 
LGTM

File: packages/flx/lib/signing.dart (hunk 1 above)
Comment (pq) [inline packages/flx/lib/signing.dart]: Yeah.  That fix just hasn't made it's way into the SDK yet.  Should be in the next roll though.

Comment (devoncarew) [inline packages/flx/lib/signing.dart]: I'm on `1.16.0-dev.3.0` (from yesterday) and am not seeing the lint reported for this case, so presumably it just landed :)

**Pull request merged**